canopycover.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python3
+"""Calculates canopy coverage for plots in georeferenced images
+"""
+
+import argparse
+import copy
+import json
+import logging
+import os
+from typing import Union
+import numpy as np
+import dateutil.parser
+from agpypeline import entrypoint, algorithm, geoimage
+from agpypeline.environment import Environment
+from osgeo import gdal, ogr
+
+from configuration import ConfigurationCanopycover
+
+
+# Array of trait names that should have array values associated with them
+TRAIT_NAME_ARRAY_VALUE = ['canopy_cover', 'site']
+
+# Mapping of default trait names to fixed values
+TRAIT_NAME_MAP = {
+    'access_level': '2',
+    'species': 'Unknown',
+    'citation_author': '"Zongyang, Li"',
+    'citation_year': '2016',
+    'citation_title': 'Maricopa Field Station Data and Metadata',
+    'method': 'Green Canopy Cover Estimation from Field Scanner RGB images'
+}
+
+
+def get_fields() -> list:
```

Comment:
* couldn't this be in the agpypeline library, and `canopy_cover` would come from the trait_name_map?

Reply:
This will be going away when Canopy Cover gets derived from rgb plot code template